src/main/java/uk/gov/pay/webhooks/deliveryqueue/managed/SendAttempter.java

```diff
@@ -70,8 +72,15 @@ public void attemptSend(WebhookDeliveryQueueEntity queueItem) {
         var retryCount = webhookDeliveryQueueDao.countFailed(queueItem.getWebhookMessageEntity());
         Instant start = instantSource.instant();
 
+        URL url = null;
```

Comment:
Can this not be added to the existing try-catch block with a new catch block to cover the `MalformedURLException` handler? We wouldn't need to explicitly define url as null

Reply:
We can't add it to the existing try-catch block as the `url` variable won't be accessible by code in the other catch blocks where it's needed.